### PR TITLE
Refresh Cache

### PR DESCRIPTION
**Pull request by dtan4**
## WHY

To refresh saved cache to new one
## WHAT

Fetch files from image and put them to cache
## TODO
- [x] Fetch files from image (running container)
- [x] Put files to cache

### CODE REVIEW

**Comment by dtan4** [comment]
Now testing the behavior with actual Docker image...

**Comment by dtan4** [comment]
@wantedly/infrastructure plz review this!!!!

**Comment by dtan4** [comment]
- [x] Create cache tarball at temporary dir, and move complete tarball to under /var
  - To avoid getting incompleted tarball at next build

**Comment by spesnova** [comment]
I'll check this later, wait for a moment.

File: docker.go [hunk 1]
```diff
@@ -51,10 +55,77 @@ func dockerBuild(build schema.Build) error {
 	return nil
 }
 
+func dockerCopy(build schema.Build) (string, error) {
```
Comment (spesnova) [inline docker.go]: I think `extractCache()` is better than `dockerCopy()`
`docker copy` is just an internal process in extracting the cache.

**Comment by spesnova** [comment]
LGTM :+1:

**Pull request merged**